package.json

```diff
@@ -6,7 +6,9 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lint:css": "npx stylelint '**/*.css'",
+    "lint:scss": "npx stylelint '**/*.scss'"
```

Comment:
Since `stylelint` is included in `devDependencies`, we can omit the `npx`. That's mainly used for cases where you want to use an npm package without installing it.

We can also delete the `lint:css` script, since we should only be using `.scss` files.